modules/project-docs/pages/sdk-full-installation.adoc

```diff
@@ -33,7 +33,7 @@ We recommend running the latest Java LTS version (i.e. at the time of writing JD
 Java 17 has various enhancements like sealed classes, pattern matching for switch expressions (in preview), and further updates and improvements on core libraries.
 
 Couchbase publishes all stable artifacts to https://search.maven.org/search?q=com.couchbase.client[Maven Central].
-The latest version (as of October 2022) is https://search.maven.org/artifact/com.couchbase.client/java-client/3.4.0/jar[3.4.0].
+The latest version (as of December 2022) is https://search.maven.org/artifact/com.couchbase.client/java-client/3.4.1/jar[3.4.1].
```

Comment:
{java-current-version} should work here - check the other SDK docs for how we do it with code and URLs

Reply:
(although perhaps best left out of dependency config if it's going to be pulled in anywhere ;) )